### ISSUE TITLE
Add to Reading List Silently Failing

### ISSUE THREAD

**Issue by pyrmont** [Ranchero-Software/NetNewsWire#4400]
## Description

Since at least iOS 18, NetNewsWire on iOS stops being able to send RSS items to Safari's Reading List. Attempting to do so will not bring up the system prompt asking if I want to allow the app to add to the Reading List but will instead fail silently. If I force quit the app and then relaunch, it starts working normally.

## Steps to Reproduce

This is difficult to reproduce as I haven't discovered what causes this problem to occur.

## Expected Result

The URL for the RSS item is added to Safari's Reading List. 

## Actual Result

The URL is not added.

**Comment by brentsimmons** [comment]
I can’t reproduce this. Using iOS 18.7.

Do you have steps to reproduce, or any idea about the conditions that might be related?

**Comment by pyrmont** [comment]
Yeah, it's been a while and I've adopted a new system for adding things to a Tab Group rather than the Reading List so am not impacted anymore (if the bug even still exists).

I'll close given it no longer seems applicable/relevant. Sorry not to have done that sooner!
